src/filters/utils.cjs

```diff
@@ -65,4 +65,29 @@ const onlyShow = (array, n) => {
   return array.slice(0, n);
 };
 
-module.exports = { typeCheck, styles, onlyShow };
+/* @docs
+label: oddNewsTags
+category: Filter
+note: |
+  Returns the matching mailchimp form tag
+  for the page where the signup form lives |
+
+params:
+  name:
+    type: string
+    note: name of page such as 'footer'
+*/
+const oddNewsTags = (name) => {
+  switch (name) {
+    case 'footer':
+      return '6264369';
+    case 'oddblog':
+      return '6265233';
+    case 'oddnews':
+      return '6265089';
```

Comment:
I'm not sure where these magic numbers come from, and the Mailchimp UX is a disaster. I do see both `oddnews-page` and `oddnews` tags in Mailchimp -- is that intentional?

Reply:
Mailchimp let's us generate sign up forms with specific tags and the only difference between the forms are these tag values. I've pulled those out from those generated forms to use here. And yes, I think originally I just had the 1 `oddnews` tag but split them out further for better tracking of where sign ups are coming from. I'll probably go in and delete that general `oddnews` one now.